add_catch(test_restore_the_array test.cpp)
